doc/source/user_guide/io.rst

```diff
@@ -1652,26 +1652,61 @@ python engine is selected explicitly using ``engine='python'``.
 Reading remote files
 ''''''''''''''''''''
 
-You can pass in a URL to a CSV file:
+You can pass in a URL to read or write remote files to many of Pandas' IO
+functions - the following example shows reading a CSV file:
 
 .. code-block:: python
 
    df = pd.read_csv('https://download.bls.gov/pub/time.series/cu/cu.item',
                     sep='\t')
 
-S3 URLs are handled as well but require installing the `S3Fs
+All URLs which are not local files or HTTP(s) are handled by
+`fsspec`_, if installed, and its various filesystem implementations
+(including Amazon S3, Google Cloud, SSH, FTP, webHDFS...).
+Some of these implementations will require additional packages to be
+installed, for example
+S3 URLs require the `s3fs
 <https://pypi.org/project/s3fs/>`_ library:
 
 .. code-block:: python
 
-   df = pd.read_csv('s3://pandas-test/tips.csv')
+   df = pd.read_json('s3://pandas-test/adatafile.json')
```

Comment:
does this work?

Reply:
There is no such (public) bucket, the previous example didn't "work" either - it's meant to demonstrate use.